src/Analyzer/ClassDescriptionBuilder.php

```diff
@@ -54,9 +57,17 @@ public function addDependency(ClassDependency $cd): self
         return $this;
     }
 
+    public function setExtends(string $FQCN, int $line): self
```

Comment:
At some point is `Extends` in other `Extend`, what do you think to unify the name?

Reply:
Thing is, extends is a reserved word and cannot be used as class name. I tried to keep consistency inside every class